controllers/configurationpolicy_controller.go

```diff
@@ -765,14 +871,17 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(policy *policyv1.Configu
 
 // getResourceAndDynamicClient creates a dynamic client to query resources and pulls the groupVersionResource
 // for an object from its mapping, as well as checking whether the resource is namespaced or cluster-level
-func getResourceAndDynamicClient(mapping *meta.RESTMapping, apiresourcelist []*metav1.APIResourceList) (dclient dynamic.Interface,
-	rsrc schema.GroupVersionResource, namespaced bool) {
+func getResourceAndDynamicClient(
+	mapping *meta.RESTMapping,
+	apiresourcelist []*metav1.APIResourceList,
```

Comment:
Optional:

The Python black auto-formatting tool would only put each parameter in a new line if they *all* could not fit on a single line. I tend to like that approach. So you could do something like this:

```suggestion
	mapping *meta.RESTMapping, apiresourcelist []*metav1.APIResourceList,
```

Reply:
I tried to go through and catch all the long function signatures... Basically to try and make it obvious where inputs end and outputs begin, and also make it obvious (by indentation) when the actual code begins.